roles/cfg_openwrt/templates/common/rc.local.j2

```diff
@@ -32,4 +32,10 @@ echo {{ override['value'] }} > {{ override['path'] }}
 {{ cmd }}
 {% endfor %}
 
+{% if snmp_devices|selectattr('snmp_profile', 'equalto', 'airos_6')|list|length >  0 %}
+# Change the option "procd_set_param stderr  1" to "0" in the file /etc/init.d/collectd
+# This is needed because airos_6 APs w/o stations resulted in a lot of local log spamming
+sed -i 's/\(procd_set_param stderr\)[[:space:]]*1/\1  0/' /etc/init.d/collectd
```

Comment:
`sed -i 's/\(procd_set_param stderr\)[[:space:]]\{1,\}1/\1 0/' /etc/init.d/collectd` would be more precise as it requires at least one space to be present.